setup.py

```diff
@@ -57,20 +57,13 @@ def find_version(*file_paths):
                     "scikit-learn>=0.24.0",
                     "wheel>=0.36.1"]
 
-DEPENDENCY_LINKS = []
-
 python_version = sys.version_info[:2]
-if python_version[0] < 3:
-    print("Only Python > 3.5 is supported")
-    sys.exit(0)
-elif python_version[1] < 6:
-    print("Only Python > 3.6 is supported")
+if python_version[0] < 3 or python_version[1] < 6:
+    print("Only Python >= 3.6 is supported")
```

Comment:
Our current README states support for >=3.7 only, should probably update this condition as well

Reply:
What reason to bump up the python version? Tensorflow 2.4 requires python>=3.6 and PyTorch 1.7 too.

Reply:
We actually validate on 3.7 only.

Reply:
Ok, since TF requires 3.6, will postpone updating setup.py.